src/NzbDrone.Core/MediaFiles/DiskScanService.cs

```diff
@@ -91,7 +91,8 @@ public void Scan(Movie movie)
 
                 if (_diskProvider.FolderEmpty(rootFolder))
                 {
-                    _logger.Warn("Movie's root folder ({0}) is empty.", rootFolder);
+                    const string template = "Movie's root folder ({0}) is empty. Protection mechanism has been triggered incase this is a filesystem mounting issue. Nothing will be updated.";
```

Comment:
why create a constant rather than simply updating the string?

Reply:
I felt like it was easier to see what was going on in the actual method call this way, otherwise the params get pushed way off the side of the screen

Reply:
Don't use a const here as it's not in keeping with the rest of the code base